dist/elasticui.js

```diff
@@ -947,14 +960,14 @@ var elasticui;
                             scope.agg_name = scope.field.replace(/[^a-z_0-9]/gmi, "_") + "_" + (directives.default_agg_count++);
                         }
                     };
-                    directive.template = '\
-            <ul class="nav nav-list" eui-aggregation="ejs.TermsAggregation(agg_name).field(field).size(size)">\
-                <li ng-repeat="bucket in aggResult.buckets">\
-                    <label eui-filter="ejs.TermsFilter(field, bucket.key)">\
-                        <span ng-if="!filter.enabled"><a href="" ng-click="filter.enabled=true">{{bucket.key}} <span class="muted">({{bucket.doc_count}})</span></a></span>\
-                        <span ng-if="filter.enabled">{{bucket.key}} <a href="" ng-click="filter.enabled=false" class="facet-remove">x</a></span>\
-                    </label>\
-                </li>\
+                    directive.template = '\
+            <ul class="nav nav-list" eui-aggregation="ejs.TermsAggregation(agg_name).field(field).size(size)">\
+                <li ng-repeat="bucket in aggResult.buckets">\
+                    <label eui-filter="ejs.TermsFilter(field, bucket.key)">\
+                        <span ng-if="!filter.enabled"><a href="" ng-click="filter.enabled=true">{{bucket.key}} <span class="muted">({{bucket.doc_count}})</span></a></span>\
+                        <span ng-if="filter.enabled">{{bucket.key}} <a href="" ng-click="filter.enabled=false" class="facet-remove">x</a></span>\
+                    </label>\
+                </li>\

```

Comment:
The only changes in this template are whitespace changes? Why was this necessary?